src/ui/public/filter_bar/filter_bar.less

```diff
@@ -49,6 +49,25 @@ filter-bar {
   }
 }
 
+.filter-panel {
+  position: relative;
+
+  .filter-panel__pill {
+    display: inline;
+  }
+
+}
+
+.filter-panel--close {
+  max-height: 36px;
+  overflow-y: hidden;
+
+  .filter-panel__pill {
+    display: inline-block;
+    width: 90%;
```

Comment:
Change to `width: calc(100% - 80px)`. At 90%, there is still the chance of overlap.